backend/app/schemas.py

```
@@ -210,8 +200,10 @@
 #     role: str
 #     content: str
 
+
 class ChatRequest(BaseModel):
-    messages: List[str] 
-    
+    messages: List[str]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)